sfdx-source/apex-mocks/test/classes/fflib_MatcherDefinitionsTest.cls

```diff
@@ -924,6 +924,20 @@ public class fflib_MatcherDefinitionsTest
             }               
 		};
 
+		list<Map<Schema.SObjectField, Object>> failingFieldValues = new list<Map<Schema.SObjectField, Object>>
+		{
+				new map<Schema.SObjectField,Object>
```

Comment:
It looks like the problem isn't tabs vs. spaces .. it's that you're using 2 tabs for each indent. If you can just change this to one tab per indent level, we're good to go

Reply:
I replaced the indentation for single tabs.